src/drivers/file/Queue.php

```diff
@@ -212,6 +212,9 @@ protected function delete($payload)
     {
         $id = $payload[0];
         $this->touchIndex(function (&$data) use ($id) {
+            if (!isset($data['reserved'])) {
+                return;
+            }
```

Comment:
should be done in another PR, let's keep this one about flock

Reply:
Yes its my error, second commit in the pull request.